### PR TITLE
Add L0 housekeeping products

### PR DESCRIPTION
**Pull request by samaloney**
* Mini report TM(3, 25), SSID 1
* Maxi report TM(3, 25), SSID 2

### CODE REVIEW

File: stixcore/products/level0/housekeeping.py [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+        control['index'] = range(len(control))
+
+        # Create array of times as dt from date_obs
+        times = [DateTime(ct, ft).as_float() for ct, ft in control['scet_coarse', 'scet_fine']]
+
+        # Data
+        data = Data()
+        data['time'] = times
+        data['sw_running'] = packets.get_value('NIXD0021', attr='value')
+        data['instrument_number'] = packets.get_value('NIXD0022', attr='value')
```
Comment (samaloney) [inline stixcore/products/level0/housekeeping.py:120]: Could probably replace this and the similar block above with a loop using a modified version of the the parameters name from the IDB? Override one that need to be modified or fix in idb

File: stixcore/products/level0/housekeeping.py [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+        data['instrument_mode'] = packets.get_value('NIXD0023', attr='value')
+        data['hk_dpu_pcb_t'] = packets.get_value('NIXD0025')
+        data['hk_dpu_fpga_t'] = packets.get_value('NIXD0026')
+        data['hk_dpu_3v3_c'] = packets.get_value('NIXD0027')
+        data['hk_dpu_2v5_c'] = packets.get_value('NIXD0028')
+        data['hk_dpu_1v5_c'] = packets.get_value('NIXD0029')
+        data['hk_dpu_spw_c'] = packets.get_value('NIXD0030')
+        data['hk_dpu_spw0_v'] = packets.get_value('NIXD0031')
+        data['hk_dpu_spw1_v'] = packets.get_value('NIXD0032')
+        data['sw_version'] = packets.get_value('NIXD0001')
```
Comment (nicHoch) [inline stixcore/products/level0/housekeeping.py:53]: i am not sure but is it possible that this values here are already converted into engineering values?

so fare we only used times and this is ok for level0.
but my understanding was that we will for this voltages.... store the the raw value in level0 and later on in level1 convert it to engineering

File: stixcore/products/level0/housekeeping.py (hunk 1 above)
Comment (nicHoch) [reply stixcore/products/level0/housekeeping.py:120]: hm yes that might be a good sugestion

**Review by nicHoch — APPROVED** [review]
did not really check for completeness of the nixes as i think the suggested generic "loop" over all HK params is a good approach.

File: stixcore/products/level0/housekeeping.py (hunk 2 above)
Comment (samaloney) [reply stixcore/products/level0/housekeeping.py:53]: Yea so with the current setup they are converted here already but as you note this isn't the plan

**Pull request merged**